### PR TITLE
Unify Rollup/Webpack templates

### PR DESCRIPTION
**Pull request by Conduitry**
My ideas and changes from #93, with a couple of tiny readme/etc tweaks. Putting this in a PR not because I think it's 100% ready, but because it's easier for people to take a look at what's changing than it was before with the completely independent repo.

When the `_template/build.sh` script is run, new `rollup` and `webpack` branches are created from the current `master`, with just the code and configuration specific to each bundler. Ideally this would be automated CI way when `master` is changed - and the CI job would have push credentials for the repo, so that it can actually create the branches.

When taking a peek at this locally, it would probably be a good idea to comment out the `git push` command in `_template/build.sh` because that WILL overwrite the branches on origin.

### CODE REVIEW

**Comment by Conduitry** [comment]
I haven't looked at the Rollup and Webpack templates for _Svelte_ recently, but if we end up with a way of this that we're happy with, it probably makes sense to use it there too.

**Comment by Conduitry** [comment]
@Rich-Harris Are you cool with this basic idea? The main drawbacks it seems to have are 1) the `master` branch is no longer directly usable, and 2) it'll require some sort of CI job with push permissions (and they'll be `push -f` in the way this is currently written, this can be adjusted if so desired).

**Comment by Conduitry** [comment]
Pushed an Extremely Important Commit to make webpack lowercase. AFAICT this is the official spelling and I didn't see any instances of 'Webpack' on their site.

Also, it sounds like we might want to handled the automated pushing to this repo with a write-capable 'deploy key'. GitHub lets you create an SSH key pair that only has access to one repo. The private half would then live in Travis config.

**Comment by Conduitry** [comment]
Got this working using Travis on another test branch, `unified-test`, which pushes to `unified-rollup` and `unified-webpack`. The new commit are made by 'Travis CI User' by default, which seems probably acceptable.

I don't know what the existing Travis or Appveyor builds were supposed to be doing - it didn't look like much.

If we're happy with how this is working, I'll bring the `unified-test` changes back into this branch (and update the branch references to use the real branches: `master`/`rollup`/`webpack`) and we can merge this finally.

File: README.md [hunk 1]
```diff
@@ -53,14 +54,14 @@ There are three simple rules for naming the files that define your routes:
...
+## Bundler config
 
-Sapper uses webpack to provide code-splitting, dynamic imports and hot module reloading, as well as compiling your Svelte components. As long as you don't do anything daft, you can edit the configuration files to add whatever loaders and plugins you'd like.
+Sapper uses Rollup or webpack to provide code-splitting and dynamic imports, as well as compiling your Svelte components. With webpack, it also provides hot module reloading. As long as you don't do anything daft, you can edit the configuration files to add whatever loaders and plugins you'd like.
 
 
 ## Production mode and deployment
 
-To start a production version of your app, run `npm run build && npm start`. This will disable hot module replacement, and activate the appropriate webpack plugins.
+To start a production version of your app, run `npm run build && npm start`. This will live relead, and activate the appropriate bundler plugins.
```
Comment (Rich-Harris) [inline README.md]: live relead?

File: README.md [hunk 2]
```diff
@@ -70,7 +71,7 @@ now
 ```
 
 
-## Using external components
+## Using external components (webpack only)
```
Comment (Rich-Harris) [inline README.md]: Someone could read this and think you can't use external components with Rollup — how about this instead?

```suggestion
## Using external components with webpack
```

**Pull request merged**